### ISSUE TITLE
Make multichoice question and answer text a textbox rather than input field

### ISSUE THREAD

**Issue by nathansherburn** [SLC3/MARS#114]
In time, we will need to add markdown and latex ([example](https://github.com/waylonflinn/markdown-it-katex)) however for now, we can just make question and answer text multi line and maintain whitespace for programming questions.

ie:

> What will this print?
> 
> for (var i = 0; i < 10; i++) {
>   console.log('hello')
> }

**Comment by akdor1154** [comment]
Currently adding quick multi-line support for poll questions and answers, with an eye out for future expansion to markdown formatting etc.

The SVG based bar chart is very cool, but multi-line on its own is difficult (should be possible but maybe difficult, and the truncation of answer text will be non-trivial to generalize to multiple lines), and further formatting (markdown, latex) will be quite difficult to implement -- with an HTML representation there is no end to the libraries we could use for formatting, but there are very few for SVG output (I can find precisely none for markdown, and only mathjax for latex).

Options:
- **multi choice answers are plain text, single line**
Basically, do nothing, and leave things as they are now. Not awful as we can still offer formatting in poll questions, just not answers. I can definitely see use-cases for latex formatting in answers though, even if markdown is massive overkill.
- **continue with SVG**
Potentially a lot of work, but we can probably at least get multiline and maybe latex working as well. This would probably require a bit of server side and api work as well, as mathjax on client is probably too slow for us to inflict on people.
- **change bar chart to use a DOM based visualization**
A fair amount of work, but means any formatting will be HTML based and thus quite easy to pick from the zoo of libraries already written. Also means that we won't be maintaining two different formatting stacks with potentially different rendering output. D3 seems to work quite well with HTML elements as well as SVG, so a full re-write shouldn't be required, though a lot of the positioning is currently SVG-only.

**Comment by nathansherburn** [comment]
Thanks for this. I ended up testing on my local machine and it looked good.

Jon and I have been discussing the larger picture of where we see MARS going and we've decided that we're going to keep questions and answers single line for now (no maintaining whitespace etc) as it opens up a can of worms both UI wise and functionality wise - if we support multiline questions then why not images, LaTeX etc.

We've decided that (at least for now) our focus will be on getting presenters to show their questions in PowerPoint (or similar) and then having a text based version of the question in MARS.

Thanks for exploring the options too!
